Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -84,32 +84,36 @@ class SectionEditorViewController: UIViewController {
     
     private func setupFocusNavigationView() {
        
-        //TODO: Localize
+        
+        let headerAccessibilityText = WMFLocalizedString("find-replace-header-accessibility", value: "Find and replace heading", comment: "Find and replace header title.")
+        let closeAccessibilityText = WMFLocalizedString("find-replace-header-close-accessibility", value: "Close find and replace", comment: "Accessibility label for closing the find and replace view.")
+        
+        focusNavigationView.configure(headerText: CommonStrings.findAndReplaceHeaderTitle, headerAccessibilityText: headerAccessibilityText, closeButtonAccessibilityText: closeAccessibilityText, traitCollection: traitCollection)
+        
         focusNavigationView.isHidden = true
-        focusNavigationView.configure(text: "Find and replace", traitCollection: traitCollection)
         focusNavigationView.delegate = self
         focusNavigationView.apply(theme: theme)
         
         focusNavigationView.translatesAutoresizingMaskIntoConstraints = false
-        navigationController?.navigationBar.addSubview(focusNavigationView)
-        
-        focusNavigationView.setNeedsLayout()
-        focusNavigationView.layoutIfNeeded()
+
+        view.addSubview(focusNavigationView)
         
-        navigationController?.navigationBar.leadingAnchor.constraint(equalTo: focusNavigationView.leadingAnchor).isActive = true
-        navigationController?.navigationBar.trailingAnchor.constraint(equalTo: focusNavigationView.trailingAnchor).isActive = true
-        focusNavigationViewBottomConstraint = navigationController?.navigationBar.bottomAnchor.constraint(equalTo: focusNavigationView.bottomAnchor, constant: focusNavigationView.frame.height)
-        focusNavigationViewBottomConstraint.isActive = true
+        view.leadingAnchor.constraint(equalTo: focusNavigationView.leadingAnchor).isActive = true
+        view.trailingAnchor.constraint(equalTo: focusNavigationView.trailingAnchor).isActive = true
+        view.safeAreaLayoutGuide.topAnchor.constraint(equalTo: focusNavigationView.topAnchor).isActive = true
```

Comment:
if these can be activated in one pass via `activateConstraints`, it might be more efficient according to https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activate

> Typically, using this method is more efficient than activating each constraint individually.

Reply:
another good idea, also looks a bit cleaner IMO. done!